include/seqan3/core/type_list/traits.hpp

```diff
@@ -329,6 +329,22 @@ auto split_after(type_list<pack1_t...>, type_list<head_t, pack2_t...>)
 template <template <typename> typename trait_t, typename ...pack_t>
 type_list<trait_t<pack_t>...> transform(type_list<pack_t...>);
 
+/*!\brief Implementation for replace_at.
+ * \tparam replace_t The type replacing the old one.
+ * \tparam pack1_t   Types in the first type list.
+ * \tparam head_t    Head of the second type list.
+ * \tparam pack2_t   Rest of the second type list.
```

Comment:
pack1 and pack2 are collections of types before and after the element subject to replacement, right? Can you make it clear in the documentation, please?